regulations/static/regulations/js/source/views/comment/comment-view.js

```diff
@@ -243,6 +249,14 @@ var CommentView = Backbone.View.extend({
     comments.add(this.model);
     this.model.save();
     this.$status.hide().html('Your comment was saved.').fadeIn();
+  },
+
+  // when "Review and Submit" is clicked, save any unsaved content
+  // then open link on button
+  submitComments: function(e) {
+    this.save();
+
+    document.location.href = window.location.origin + $(e.target).attr('href');
```

Comment:
This feels wonky -- we're preventing the event from completing, sending an event, and then "manually" completing the same action. I know a lot of this indirection stems from Backbone's architecture, but is there a more direct way to accomplish what we need?